src/adapters/Cornerstone3D/EllipticalROI.js

```diff
@@ -0,0 +1,167 @@
+import MeasurementReport from "./MeasurementReport";
+import TID300Ellipse from "../../utilities/TID300/Ellipse";
+import CORNERSTONE_3D_TAG from "./cornerstone3DTag";
+
+const ELLIPTICALROI = "EllipticalROI";
+const FINDING = "121071";
+const FINDING_SITE = "G-C0E3";
+
+const trackingIdentifierTextValue = "Cornerstone3DTools@^0.1.0:EllipticalROI";
+
+class EllipticalROI {
+    constructor() {}
+
+    // TODO: this function is required for all Cornerstone Tool Adapters, since it is called by MeasurementReport.
```

Comment:
Shall we remove this TODO? Or at least write that a base class with an abstract method would be cleaner?

Reply:
done